docs/en/docs/advanced/compute_node.md

```diff
@@ -96,15 +96,17 @@ HeartbeatFailureCounter: 0
 ```
 
 ### Usage
-When using the [MultiCatalog](../lakehouse/multi-catalog/multi-catalog) , the query will be preferentially scheduled to the compute node.
+When using the [MultiCatalog](https://doris.apache.org/docs/dev/lakehouse/multi-catalog/) , the query will be preferentially scheduled to the compute node.
```

Comment:
It seems had been fix by #16284 but the doc is not released

Reply:
ok, i revert this code